cluster/gce/util.sh

```diff
@@ -869,6 +853,30 @@ function kube-push {
   echo
 }
 
+function kube-update-nodes() {
+  action=${1}
+
+  OUTPUT=${KUBE_ROOT}/_output/logs
+  mkdir -p ${OUTPUT}
+
+  echo "Updating node metadata... "
+  write-node-env
+  for (( i=0; i<${#MINION_NAMES[@]}; i++)); do
+    add-instance-metadata-from-file "${MINION_NAMES[$i]}" "kube-env=${KUBE_TEMP}/node-kube-env.yaml" &
```

Comment:
I think before you head down that route, we should talk about just doing it on the node, watching the instance metadat. But sure, it's a fine todo.